internal/cmd/controller/gitops/reconciler/gitjob_controller.go

```diff
@@ -141,6 +152,67 @@ func generationOrCommitChangedPredicate() predicate.Predicate {
 	}
 }
 
+func (r *GitJobReconciler) createJobRBAC(ctx context.Context, gitrepo *v1alpha1.GitRepo) error {
+	// No update needed, values are the same. So we ignore AlreadyExists.
+	saName := name.SafeConcatName("git", gitrepo.Name)
+	sa := grutil.NewServiceAccount(gitrepo.Namespace, saName)
+	if err := controllerutil.SetControllerReference(gitrepo, sa, r.Scheme); err != nil {
+		return err
+	}
+	if err := r.Create(ctx, sa); err != nil && !errors.IsAlreadyExists(err) {
+		return err
+	}
+
+	role := grutil.NewRole(gitrepo.Namespace, saName)
+	if err := controllerutil.SetControllerReference(gitrepo, role, r.Scheme); err != nil {
+		return err
+	}
+	if err := r.Create(ctx, role); err != nil && !errors.IsAlreadyExists(err) {
+		return err
+	}
+
+	rb := grutil.NewRoleBinding(gitrepo.Namespace, saName)
+	if err := controllerutil.SetControllerReference(gitrepo, rb, r.Scheme); err != nil {
+		return err
+	}
+	if err := r.Create(ctx, rb); err != nil && !errors.IsAlreadyExists(err) {
+		return err
+	}
+
+	return nil
+}
+
+func (r *GitJobReconciler) createTargetsConfigMap(ctx context.Context, gitrepo *v1alpha1.GitRepo) error {
+	configMap, err := grutil.NewTargetsConfigMap(gitrepo)
+	if err != nil {
+		return err
+	}
+	if err := controllerutil.SetControllerReference(gitrepo, configMap, r.Scheme); err != nil {
+		return err
+	}
+	data := configMap.BinaryData
+	_, err = controllerutil.CreateOrUpdate(ctx, r.Client, configMap, func() error {
+		configMap.BinaryData = data
+		return nil
+	})
+
+	return err
+}
+
+func (r *GitJobReconciler) validateExternalSecretExist(ctx context.Context, gitrepo *v1alpha1.GitRepo) error {
+	// Validate external secrets exist
```

Comment:
Is this comment still necessary, given the name of the method? ;)

Reply:
:bowtie: good point. comment deleted